tests/test_utils.py

```diff
@@ -93,16 +95,20 @@ def test_client_authenticated(app):
 
 
 def test_create_test_user(app):
-    """Test for testutils.py:create_test_user().
+    """Tests for testutils.py:create_test_user().
 
     Demonstrates basic usage and context requirements."""
     ext = InvenioAccounts(app)
     email = 'test@test.org'
     password = '1234'
 
-    # Can't access the datastore outside of an application context
-    with pytest.raises(RuntimeError):
-        testutils.create_test_user(email, password)
+    # If pytest-flask isn't available we can't access the datastore outside of
```

Comment:
Only reason I can think of is as a helpful reminder to future developers who might not be aware of why this repo's tests are different than the ones in repos not using pytest-flask. But this probably isn't the place to include a tidbit like that.